polkadot/node/network/statement-distribution/src/v2/mod.rs

```diff
@@ -681,14 +681,23 @@ pub(crate) async fn handle_active_leaves_update<Context>(
 				.map_err(JfyiError::FetchValidatorGroups)?
 				.1;
 
+		let maybe_claim_queue = fetch_claim_queue(ctx.sender(), new_relay_parent)
```

Comment:
now we are fetching it twice, first time here and also in `determine_groups_per_para`

Reply:
Fixed